apps/src/gamelab/GameLab.js

```diff
@@ -265,7 +265,12 @@ GameLab.prototype.init = function (config) {
   config.centerEmbedded = false;
   config.wireframeShare = true;
   config.responsiveEmbedded = true;
-  config.noHowItWorks = true;
+  if (!config.droplet) {
+    config.noHowItWorks = false;
```

Comment:
Nice update!
Nit: To make this neater, we could shorten this conditional to: `config.noHowItWorks = config.droplet;`

Reply:
So true! will make that change now then push.